src/typing_extensions.py

```diff
@@ -1543,7 +1378,7 @@ def get_origin(tp):
         """
         if isinstance(tp, _AnnotatedAlias):
```

Comment:
I think we may as well just use the fully qualified form, and then we can get rid of `from typing import _AnnotatedAlias, _BaseGenericAlias` statement a few lines above altogether 

```suggestion
        if isinstance(tp, typing._AnnotatedAlias):
```